pkg/proxy/nftables/proxier.go

```diff
@@ -399,6 +401,19 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 		Comment: ptr.To("rules for kube-proxy"),
 	})
 
+	// Do an extra "add+delete" once to ensure all previous base chains in the table
+	// will be recreated. Otherwise, altering properties (e.g. priority) of these
+	// chains would fail the transaction.
+	proxier.flushOnce.Do(func() {
+		for _, bc := range nftablesBaseChains {
+			chain := &knftables.Chain{
+				Name: bc.name,
+			}
+			tx.Add(chain)
+			tx.Delete(chain)
```

Comment:
Hm... it looks like this works in the e2e test, which is a little surprising to me... I would have expected you'd get back `EBUSY` while trying to delete them.

I guess these chains refer to _other_ chains, but nothing refers to them. OK.

Reply:
Yes, I checked base chains are not referenced.